scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/utils/VariablesMap.java

```diff
@@ -31,6 +31,8 @@
 import java.util.Map;
 import java.util.Set;
 
+import org.ow2.proactive.scheduler.task.context.TaskContextVariableExtractor;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.ow2.proactive.scheduler.task.context.TaskContextVariableExtractor'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)